app/scripts/modules/openstack/serverGroup/configure/wizard/access/accessSettings.html

```diff
@@ -28,6 +28,36 @@
                                             required="true"
                                             model="command.securityGroups"></os-app-cache-backed-multi-select-field>
 
+    <div class="form-group">
+      <div class="col-md-5 sm-label-left">
+        <b>Associate Floating IP Address</b>
+        <help-field key="openstack.network.floatingip"></help-field>
+      </div>
+      <div class="col-md-2 radio">
+        <label>
+          <input type="radio" ng-model="command.associatePublicIpAddress" ng-value="true" id="associatePublicIpAddressTrue"/>
```

Comment:
Are these `id`s doing anything? They are a little generically named, and it would be unfortunate if anyone came up with a matching CSS rule down the road.